cmake/InstallLayout.cmake

```diff
@@ -79,6 +79,20 @@ function(install_symlink)
   endif()
 endfunction()
 
+function(install_package_symlink)
```

Comment:
Can you please rename this function? It is very confusing to me having a function name prefixed with `install_` even though it doesn't actually install anything...